### ISSUE TITLE
Series "highlight" can't be switched off for LineChart

### ISSUE THREAD

**Issue by ipeshev** [telerik/kendo-angular#3223]
**Describe the bug**
When type of series is "line" , 
 <kendo-chart-series-defaults [highlight]="{ visible: false }">
      </kendo-chart-series-defaults> 
does not actually switch off highlight of inactive series.

**To Reproduce**
https://stackblitz.com/edit/angular-ssnh4p-bxwtuk?file=app/app.component.ts

**Expected behavior**
It works for type "column" for example https://stackblitz.com/edit/angular-2ppjfk?file=app/app.component.ts

**Comment by TrAx1988** [comment]
Got the same problem...

**Comment by kennfarbe** [comment]
Hey. What is with this bug?

**Comment by dimitar-pechev** [comment]
Can be worked-around by setting the `inactiveOpacity` value of the [`highlight`](https://www.telerik.com/kendo-angular-ui/components/charts/api/SeriesHighlight/#toc-inactiveopacity) prop to `1`.

https://stackblitz.com/edit/angular-ssnh4p-1j3b9z

```
      <kendo-chart-series-defaults
           type="line"
           [highlight]="{ inactiveOpacity: 1 }"
      >     
      </kendo-chart-series-defaults>
```

**Comment by dimitar-pechev** [comment]
Fix available in `@progress/kendo-angular-charts@5.4.0`. Still undergoing some internal QA, but can be tested and used as of now:

https://stackblitz.com/edit/angular-ssnh4p-ya3he2

**Comment by ipeshev** [comment]
Fix shipped with @progress/kendo-angular-charts@5.4.0.
